### ISSUE TITLE
Tooltips: A new Criteria?

### ISSUE THREAD

**Issue by frankelavsky** [Chartability/POUR-CAF#17]
This is somewhat related to the already-open #8, but should we have a criteria related to tooltips?

I often talk about Sarah Higley's [tooltips in WCAG 2.1 article](https://sarahmhigley.com/writing/tooltips-in-wcag-21/) for existing criteria like "actions are not forgivable" and "target pointer size is too small" but I really think that visualization folks just need some really specific guidance on how to make tooltips better.

The reality is that we might not need tooltip criteria in Chartability. But we _definitely_ need someone to make an article about how to make accessible tooltips for data visualizations, since it is such a common issue related to a very common practice in our line of work.

That being said, I kinda want criteria specific to tooltips since I've brought them up so much over the years.... I'm open to being convinced either way. Chartability tends to focus more on broader principles of our practice as a whole in interactive visualization, but it's also okay to have specific criteria too (like in the case of adding data tables).

(Thanks to extended discussions with @shelbyelavsky on this.)

**Comment by emilykund** [comment]
I'm in the clarify expectations camp. WCAG has the success criteria & then techniques to help meet the SCs. 

I think specifying actions/guidelines on how we can achieve compliance using tooltips will be helpful because you have people who are SMEs in visualizing data but not in accessibility or the underpinnings of the tool if they're using a commercial product.

**Comment by frankelavsky** [comment]
Closing for lack of further discussion. I don't think we need to add this in right now.
